entrypoint.py

```diff
@@ -62,23 +64,41 @@
 git.config('--global', 'user.name', os.environ['INPUT_GIT_USER_NAME'])
 git.config('--global', 'user.email', os.environ['INPUT_GIT_USER_EMAIL'])
 
-release_notes = ""
-for rnf in release_notes_files:
-  release_notes += open(rnf, 'r').read().rstrip() + '\n\n'
-next_release_notes = open('.release-notes/next-release.md', 'a+')
-next_release_notes.write(release_notes)
-next_release_notes.close()
-
-print(INFO + "Adding git changes." + ENDC)
-for rnf in release_notes_files:
-  git.rm(rnf)
-git.add('.release-notes/next-release.md')
-git.commit('-m', "Updating release notes for PR #" + str(pr_id)+ " [skip-ci]")
-
-print(INFO + "Pulling latest changes." + ENDC)
-git.pull()
-
-print(INFO + "Pushing updated release notes." + ENDC)
+# check to make sure that the PR had a changelog label
+# if it didn't delete the release notes file(s) and exit.
+pull_request = repo.get_pull(pr_id)
+found_changelog_label = False
+for l in pull_request.labels:
+  print(INFO + "PR had label: " + l.name + ENDC)
+  try:
+    changelog_labels.index(l.name)
+    found_changelog_label = True
+    break
+  except:
+    None
```

Comment:
Can't say I've seen `index` used this way before. But it seems to check out 😄 

The following code might be clearer:

```suggestion
  if l.name in changelog_labels:
    found_changelog_label = True
    break
```